flink/src/main/java/org/apache/iceberg/flink/sink/FlinkSink.java

```diff
@@ -33,7 +33,7 @@
 import org.apache.flink.table.api.TableSchema;
 import org.apache.flink.table.data.RowData;
 import org.apache.flink.table.data.util.DataFormatConverters;
-import org.apache.flink.table.runtime.typeutils.RowDataTypeInfo;
+import org.apache.flink.table.runtime.typeutils.InternalTypeInfo;
```

Comment:
As this `InternalTypeInfo` is the flink internal API that we have to depends on in the flink+iceberg integration work,  I'd like to move this `InternalTypeInfo.of(RowType rowType)` into a separate small utility methods,  in case of changing every places that refer this internal class. 

https://github.com/openinx/incubator-iceberg/commit/2a635719955f3eecd92685cb8f5486c46aa60095#diff-e4d1084ea160d030f3a225948d224678436b79eb5a94df230c71cdc7327f99d8R43

Reply:
will do assuming you meant just consolidation of calls to Flink Internal and PublicEvolve APIs. we don't need to support both Flink 1.11 and 1.12 in the compatibility util class.